controller/test-files/root/list/ok_root.res.payload.golden.json

```diff
@@ -0,0 +1,140 @@
+{
+  "data": {
+    "id": "00000000-0000-0000-0000-000000000001",
+    "links": {
+      "self": "http:///api"
+    },
+    "relationships": {
+      "codebases_che_start": {
+        "links": {
+          "related": "/codebases/che/start"
+        }
+      },
+      "codebases_che_state": {
+        "links": {
+          "related": "/codebases/che/state"
+        }
+      },
+      "current_user": {
+        "links": {
+          "related": "/user"
+        }
+      },
+      "deployments_environments": {
+        "links": {
+          "related": "/deployments/environments"
+        }
+      },
+      "features": {
+        "links": {
+          "related": "/features"
+        }
+      },
+      "filters": {
+        "links": {
+          "related": "/filters"
+        }
+      },
+      "login_authorize": {
+        "links": {
+          "related": "/login/authorize"
+        }
+      },
+      "login_generate": {
+        "links": {
+          "related": "/login/generate"
+        }
+      },
+      "login_refresh": {
+        "links": {
+          "related": "/login/refresh"
+        }
+      },
+      "logout": {
+        "links": {
+          "related": "/logout"
+        }
+      },
+      "render": {
+        "links": {
+          "related": "/render"
+        }
+      },
+      "root": {
+        "links": {
+          "related": "/root"
+        }
+      },
+      "search": {
+        "links": {
+          "related": "/search"
+        }
+      },
+      "search_codebases": {
+        "links": {
+          "related": "/search/codebases"
+        }
+      },
+      "search_spaces": {
+        "links": {
+          "related": "/search/spaces"
+        }
+      },
+      "search_users": {
+        "links": {
+          "related": "/search/users"
+        }
+      },
+      "spaces": {
+        "links": {
+          "related": "/spaces"
+        }
+      },
+      "spacetemplates": {
+        "links": {
+          "related": "/spacetemplates"
+        }
+      },
+      "status": {
+        "links": {
+          "related": "/status"
+        }
+      },
+      "trackerqueries": {
+        "links": {
+          "related": "/trackerqueries"
+        }
+      },
+      "trackers": {
+        "links": {
+          "related": "/trackers"
+        }
+      },
+      "user_services": {
+        "links": {
+          "related": "/user/services"
+        }
+      },
+      "users": {
+        "links": {
+          "related": "/users"
+        }
+      },
+      "userspace_*": {
```

Comment:
Is it on purpose to have a `*` in the link name? According to the JSONAPI spec, this is not allowed: http://jsonapi.org/format/#document-member-names

Reply:
We have an endpoint which contains an asterisk in it: 
`/userspace/*`

Reply:
We should really limit member names to `[a-zA-Z0-9]` and get rid of all the other characters. Would that be cool for you?

Reply:
We could do that! I also spoke with Aslak about this. He mentioned that this endpoint was actually a hack and can be removed now - perhaps we can remove it instead?